posts/1517766794846-what-is-prototype-and-how-to-eat-it-in-a-right-way.md

```diff
@@ -0,0 +1,79 @@
+Author: Alex Pletnov
+Title: What is Prototype and How to Eat it in a Right Way
+Subtitle: What is Prototype and How to Eat it in a Right Way
+Meta title: What is Prototype and How to Eat it in a Right Way
+Meta description: What is Prototype and How to Eat it in a Right Way
+Preview image: /static/posts/prototype-preview.png
+
+![Heroes](/static/posts/prototype-inarticle.png)
+
+Has it ever occured to you that you see a familiar word but it feels like it has a different meaning in certain cases?
+
+![Heroes](/static/posts/prototype-inartical2.jpg)
```

Comment:
`article` not `artical`. Also, please name images after their content, like `pree-device-in-hand`